src/App.tsx

```diff
@@ -1,24 +1,365 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import cn from 'classnames';
+import {
+  SetStateAction,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './Components/Header';
+import { Section } from './Components/Section';
+import { Footer } from './Components/Footer';
+import { Todo } from './types/Todo';
+import * as todoService from './api/todos';
+import { Status } from './types/Status';
 
-const USER_ID = 0;
+const USER_ID = 12083;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [erroMessage, setErrorMessage] = useState('');
+  const [statusTodo, setStatusTodo] = useState('');
+  const [tempTodos, setTempTodos] = useState<Todo[] | null>(null);
+  const [isLoad, setIsLoad] = useState(false);
+
+  const myInputRef = useRef<HTMLInputElement>(null);
+
+  const handleClearCompleted = () => {
+    const completeTodos = todos.filter(todo => todo.completed);
+
+    setIsLoad(true);
+    setTempTodos(completeTodos);
+
+    Promise.all(completeTodos.map(delTodo => {
+      return todoService.deleteTodos(delTodo.id)
```

Comment:
but this function deleting a todo, not todos, right?
```suggestion
      return todoService.deleteTodo(delTodo.id)
```